server/tables/tables.go

```diff
@@ -173,6 +174,7 @@ type Invocation struct {
 	Tags string
 
 	ParentInvocationID string `gorm:"index:parent_invocation_id_index"`
```

Comment:
should we remove this index on parent_invocation_id ?

Reply:
Yeah I'll clean this up separately. Do you think it's still worth keeping that field? We could derive it from joining on ParentRunID instead

Reply:
hm I guess we use this for rendering the parent workflow invocation ID in the breadcrumb, so we might want to keep it for now.

Reply:
No I don't think we use this field anywhere because I just added it. In the breadcrumb, I think we use the fact that the parent invocation ID is in the build metadata

We don't have a specific use case for it now, so I'll just delete the field and the index